omod/src/main/webapp/resources/scripts/conditionlist/controllers/condition.controller.js

```diff
@@ -8,7 +8,7 @@
         var onsetDatePicker;
         var startDateElement;
         var endDatePicker;
-        var endDateElement
+        var endDateElement;
```

Comment:
My build started failing and I tracked it down to lots of missing semi-colons.  It also didn't like the use of `let`, `const`, or other ES6+ syntaxes I tried to use.

Reply:
Yeah, we build this with a very old version of NodeJS. I don't think `let` / `const` was supported until Node 8 and other bits (`import`) were much, much later.